stacktrace dump on `brig build list` with no builds

I have a project that had an errored out build. `brig run -f brigade.js bacongobbler/brigade-charts` never started a worker pod for some reason. However, because of this, when I run `brig build list bacongobbler/brigage-charts` now, I see this:

```

><> brig project list
NAME                            ID                                                              REPO
bacongobbler/brigade-charts     brigade-90e598a16b77cbcff628e99c6610d03a373602f144cf70ea3bf957  github.com/bacongobbler/brigade-charts
><> brig build list bacongobbler/brigade-charts
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0x1a8acf9]

goroutine 1 [running]:
github.com/Azure/brigade/brig/cmd/brig/commands.listBuilds(0x1da12e0, 0xc4200d6008, 0x7ffeefbffb50, 0x1b, 0x1b1c560, 0x23cd640)
        /Users/bacongobbler/code/go/src/github.com/Azure/brigade/brig/cmd/brig/commands/build_list.go:73 +0x249
github.com/Azure/brigade/brig/cmd/brig/commands.glob..func2(0x2419860, 0xc42028d280, 0x1, 0x1, 0x0, 0x0)
        /Users/bacongobbler/code/go/src/github.com/Azure/brigade/brig/cmd/brig/commands/build_list.go:35 +0x7b
github.com/Azure/brigade/vendor/github.com/spf13/cobra.(*Command).execute(0x2419860, 0xc42028d250, 0x1, 0x1, 0x2419860, 0xc42028d250)
        /Users/bacongobbler/code/go/src/github.com/Azure/brigade/vendor/github.com/spf13/cobra/command.go:599 +0x3db
github.com/Azure/brigade/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x2419200, 0x1, 0xc42017ff40, 0x1a9389e)
        /Users/bacongobbler/code/go/src/github.com/Azure/brigade/vendor/github.com/spf13/cobra/command.go:689 +0x2d4
github.com/Azure/brigade/vendor/github.com/spf13/cobra.(*Command).Execute(0x2419200, 0xc4200b4058, 0x0)
        /Users/bacongobbler/code/go/src/github.com/Azure/brigade/vendor/github.com/spf13/cobra/command.go:648 +0x2b
main.main()
        /Users/bacongobbler/code/go/src/github.com/Azure/brigade/brig/cmd/brig/main.go:11 +0x31
```

Re-running a new job with `brig run` seemed to have resolved this. Perhaps it was just an intermittent scheduling error for the cluster?

--- a/brig/cmd/brig/commands/build_list.go
+++ b/brig/cmd/brig/commands/build_list.go
@@ -67,10 +67,11 @@ func listBuilds(out io.Writer, project string) error {
 
 	for _, b := range bs {
 		status := "???"
+		since := "???"
 		if b.Worker != nil {
 			status = b.Worker.Status.String()
+			since = duration.ShortHumanDuration(time.Since(b.Worker.EndTime))
 		}
-		since := duration.ShortHumanDuration(time.Since(b.Worker.EndTime))
 		table.AddRow(b.ID, b.Type, b.Provider, b.ProjectID, status, since)
 	}
 	fmt.Fprintln(out, table)

